README.md

```diff
@@ -462,7 +464,7 @@ If you have a problem or suggestion, please [open an issue](https://github.com/o
   - [`doddo/octodns-lexicon`](https://github.com/doddo/octodns-lexicon): Use [Lexicon](https://github.com/AnalogJ/lexicon) providers as octoDNS providers.
   - [`asyncon/octoblox`](https://github.com/asyncon/octoblox): [Infoblox](https://www.infoblox.com/) provider.
   - [`sukiyaki/octodns-netbox`](https://github.com/sukiyaki/octodns-netbox): [NetBox](https://github.com/netbox-community/netbox) source.
-  - [`jcollie/octodns-netbox-dns`](https://github.com/jcollie/octodns-netbox-dns): [NetBox-DNS Plugin](https://github.com/auroraresearchlab/netbox-dns) provider.
+  - [`olofvndrhr/octodns-netbox-dns`](https://github.com/olofvndrhr/octodns-netbox-dns): [NetBox-DNS Plugin](https://github.com/peteeckel/netbox-plugin-dns) provider.
```

Comment:
Looks like @jcollie is still active on GitHub in other projects even though the octodns-netbox-dns is not to give them a chance to speak up just in case. I see the pypi version has your username so if after a bit if we haven't heard one way or another I'll let this go through.